imports/plugins/included/product-variant/components/productGridItems.js

```diff
@@ -66,9 +66,7 @@ class ProductGridItems extends Component {
       if (this.props.isMediumWeight()) {
         return (
           <div className={`product-additional-images ${this.renderVisible()}`}>
-            {this.props.additionalMedia().map((media) => {
-              return <span key={media._id} className="product-image" style={{ backgroundImage: `url('${media.url({ store: "medium" })}')` }} />;
-            })}
+            {this.props.additionalMedia().map((media) => <span key={media._id} className="product-image" style={{ backgroundImage: `url('${media.url({ store: "medium" })}')` }} />)} // eslint-disable-line max-len
```

Comment:
I'm still seeing a `max-len` error for this line when I run `eslint .`

Reply:
This is fixed in Part 2 of this PR: https://github.com/reactioncommerce/reaction/pull/3578/files#diff-a25b0ba02ae71bebe4fe3574e45e01f8R69